grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -1095,7 +1095,7 @@ class StockMovementService {
         shipmentService.sendShipment(shipments[0], null, user, requisition.origin, stockMovement.dateShipped ?: new Date())
 
         // Create temporary receiving area for the Partial Receipt process
-        if (grailsApplication.config.openboxes.receiving.createReceivingLocation.enabled) {
+        if (grailsApplication.config.openboxes.receiving.createReceivingLocation.enabled && stockMovement.destination.hasBinLocationSupport()) {
```

Comment:
I'm not sure if this is correct, but I think this bin is used only on partial receiving page, right?